src/api/internal/io/HostAddress_p.cpp

```diff
@@ -322,11 +322,12 @@ std::string HostAddress::GetIPString(void) const {
     // IPv6 format
     else if ( m_protocol == HostAddress::IPv6Protocol ) {
         for ( uint8_t i = 0; i < 8; ++i ) {
-            if ( i != 0 )
+            if ( i != 0 ) {
                 ss << ':';
                 ss << hex << ( (uint16_t(m_ip6Address[2*i]) << 8) |
                                (uint16_t(m_ip6Address[2*i+1]))
                              );
+            }
```

Comment:
@pezmaster31 please check this scope, I wasn't sure whether the indentation was indeed incorrect.

Reply:
The code is correct, the indentation is not. The if clause is there to skip a leading colon.